Parsers/ASimProcessEvent/Parsers/ASimProcessCreateTrendMicroVisionOne.yaml

```diff
@@ -0,0 +1,156 @@
+Parser:
+  Title: Process Create ASIM parser
+  Version: '0.1.0'
+  LastUpdated: Jul 17, 2023
+Product:
+  Name: Trend Micro Vision One
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: Trend Micro Vision One documentation
+  Link:
+    https://docs.trendmicro.com/en-us/enterprise/trend-vision-one/xdr-part/search-app/data-mapping-intro/data-mapping-detecti.aspx
+    https://docs.trendmicro.com/en-us/enterprise/trend-vision-one/xdr-part/search-app/data-mapping-intro/data-mapping-endpoin.aspx
+    https://automation.trendmicro.com/xdr/api-v3#tag/Observed-Attack-Techniques-Pipeline/paths/~1v3.0~1oat~1dataPipelines~1%7Bid%7D~1packages~1%7BpackageId%7D/get
+    https://automation.trendmicro.com/xdr/api-v3#tag/Observed-Attack-Techniques/paths/~1v3.0~1oat~1detections/get
+Description: |
+  This ASIM parser supports normalizing Trend Micro Vision One logs to the ASIM Process Create normalized schema. Trend Micro Vision One events are captured through Trend Vision One data connector which ingests XDR logs into Microsoft Sentinel through the Trend Vision One API.
+ParserName: ASimProcessCreateTrendMicroVisionOne
+EquivalentBuiltInParser: _ASim_ProcessEvent_CreateTrendMicroVisionOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let GetFilenamePart = (path: string) { tostring(split(path, @'\')[-1]) };
+  let EventSeverityLookup = datatable(detail_filterRiskLevel_s: string, EventSeverity: string)[
+      "low", "Low",
+      "medium", "Medium",
+      "high", "High",
+      "info", "Informational",
+      "critical", "High"
+  ];
+  let IntegrityLevelLookup = datatable(IntegrityLevel: real, IntegrityType: string)
+      [
+      0, "Untrusted",
+      4096, "Low",
+      8192, "Medium",
+      12288, "High",
+      16384, "System"
+  ];
+  let parser = (disabled: bool=false) {
+      TrendMicro_XDR_OAT_CL
+      | where not(disabled)
+          and detail_eventId_s == "TELEMETRY_PROCESS"
+          and detail_eventSubId_s == "TELEMETRY_PROCESS_CREATE"
+      | parse filters_s with * "[" filters: string "]"
+      | parse-kv filters as (description: string, name: string) with (pair_delimiter=",", kv_delimiter=":", quote='"')
+      | extend
+          ActingProcessId = tostring(toint(detail_processPid_d)),
+          TargetProcessId = tostring(toint(detail_objectPid_d)),
+          ParentProcessId = tostring(toint(detail_parentPid_d)),
```

Comment:
We are getting value with type Real(Double). Suppose if we directly converting it to string then it show value with decimal(Like, 1.0 etc...) that is why we converting it to int and then string.

Reply:
In logs, we are getting values like 14240 but the column type is real hence when we directly convert it to string, it becomes 14240.0
To avoid this we are first converting it to int and then string.